ptsprojects/zephyr/sm.py

```diff
@@ -115,6 +115,7 @@ def test_cases(pts):
     pts_bd_addr = pts.q_bd_addr
 
     stack = get_stack()
+    stack.gap_init()
```

Comment:
I see that `stack.gap_init()` is part of the `pre_conditions`.
Is there any reason we want to call it here/twice?

Reply:
The same question to me. The others look good.

Reply:
stack.gap.oob_legacy is called before pre_conditions which sets gap.init. If I do not set it there, we get the following:
AttributeError: 'NoneType' object has no attribute 'oob_legacy'

From what I see from mynewt, they initialize gap as the main function and don't put in preconditions. We can do a similar trick.